static/app/views/performance/browser/webVitals/webVitalsLandingPage.tsx

```diff
@@ -73,6 +75,14 @@ export function WebVitalsLandingPage() {
   const fidDeprecationTimestampString =
     moment(FID_DEPRECATION_DATE).format('DD MMMM YYYY');
 
+  useHasDataTrackAnalytics(
+    new MutableSearch(
+      'span.op:[ui.interaction.click,ui.interaction.hover,ui.interaction.drag,ui.interaction.press]'
```

Comment:
A more exhaustive check on this would be to check for all web vitals rather than just INP, since INP is not supported on all browsers yet. You could do this via the `has` check on all these measurement tags, i.e `has:measurements.cls OR has:measurement.fcp OR has:measurements.inp OR has:measurements.fid OR has:measurements.lcp ...`

![image](https://github.com/getsentry/sentry/assets/16740047/fe46d038-8af5-4d93-a181-0060cef3d6bd)

Reply:
No option to filter by `has:..` and OR conditionals in useSpanMetrics, so deciding to stick with the INP interactions. INP interactions are supported by all Chromium-browsers; if the user receives at least 1 interaction from a Chrome browser, then the metrics search will return a positive count.